parts/dcosagentcustomscript.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+set -e
+
+update_attribute()
+{
+    if [ ! -a /etc/mesosphere/roles/master ];
+    then
+        echo adding FD / UD attributes
+        meta=$( curl http://169.254.169.254/metadata/v1/InstanceInfo )
+        # parsing with bash to avoid jq install. This might break when the metadata service changes

```

Comment:
I was indeed trying avoid the jq install on the agent since it's not installed by default. I had a jq implementation, then decided against it as you see from the comment. I like the regex approach, too. I'll commit the change shortly